packages/edit-site/src/components/layout/index.js

```diff
@@ -172,61 +172,61 @@ export default function Layout( { route } ) {
 
 					<EditorSnackbars />
 
-					{ isMobileViewport && areas.mobile && (
-						<div className="edit-site-layout__mobile">
-							{ canvasMode !== 'edit' && (
-								<SidebarContent routeKey={ routeKey }>
-									<SiteHubMobile
-										ref={ toggleRef }
-										isTransparent={
-											isResizableFrameOversized
-										}
-									/>
-								</SidebarContent>
+					<ErrorBoundary>
```

Comment:
This is still using a single ErrorBoundary for the whole app, I was suggesting one around "area.mobile", one around "area.sidebar", one around "area.conten" and another one around "area.preview"

Reply:
The idea is that if something breaks in any of these areas, you can continue to use the other areas.

Reply:
Got it. I'll update this properly tomorrow. Thanks, Riad!